reana/config.py

```diff
@@ -13,6 +13,11 @@
     "reana-demo-root6-roofit",
     "reana-demo-worldpopulation",
     "reana-demo-atlas-recast",
+    "reana-demo-bsm-search",
+    "reana-demo-cms-h4l",
+    "reana-demo-cms-dimuon-mass-spectrum",
+    "reana-demo-alice-pt-analysis",
+    "reana-demo-alice-lego-train-test-run",
```

Comment:
You may want to remove them from `REANA_LIST_DEMO_ALL` below so that they won't appear twice there.